### DISCUSSION TITLE
Sync db data for all users

### DISCUSSION THREAD

**Discussion by Amonlol** [nocodb/nocodb#3960] [Q&A]
So let's say I have two people who are currently working at the same table. One of them changed a field in the table. The only way for the other person to see this is to refresh the page or open that table again. Is there a way to automatically force data update for every person who uses the same table in parallel to sync data (via API and\or webhooks or else)?

Thanks!

**Comment by dstala** [comment]
Real-time capabilities are not available yet. I can't think of any simple alternative using API or Webhooks

**Reply by Amonlol** [reply]
Is it possible to use post\get url requests to refresh pages of all users currently working in one table after data modifications?
